pkg/cli/common.go

```diff
@@ -220,6 +226,45 @@ func GetFromAndBudFlags(flags *FromAndBudResults, usernsResults *UserNSResults,
 	return fs
 }
 
+func SetDefaultConfig(cmd *cobra.Command, defaultConfig *config.Config) error {
```

Comment:
Do not need the function `SetDefaultConfig` because gloabl option `--containers-conf` was removed.

Reply:
Removed